src/EnergyPlus/WeatherManager.cc

```diff
@@ -6957,7 +6954,8 @@ namespace Weather {
         }
     }
 
-    Real64 WaterMainsTempFromCorrelation(EnergyPlusData &state, Real64 const AnnualOAAvgDryBulbTemp, Real64 const MonthlyOAAvgDryBulbTempMaxDiff)
+    Real64
+    WaterMainsTempFromCorrelation(EnergyPlusData const &state, Real64 const AnnualOAAvgDryBulbTemp, Real64 const MonthlyOAAvgDryBulbTempMaxDiff)
```

Comment:
[src/EnergyPlus/WeatherManager.cc:6960]:(style),[constParameterReference],Parameter 'state' can be declared as reference to const